release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -242,6 +250,16 @@ With this release, if a user manually modifies a pod security admission label fr
 
 For more information, see xref:../authentication/understanding-and-managing-pod-security-admission.adoc#security-context-constraints-psa-sync-exclusions_understanding-and-managing-pod-security-admission[Pod security admission synchronization namespace exclusions].
 
+[id="ocp-4-14-auth-cco-sts"]
+==== Authentication mode detection and Operator support for cloud providers: AWS STS
+
+With this release, the Cloud Credential Operator (CCO) can detect when it is running on clusters enabled to use short-term credentials on certain cloud providers. This release adds support for detecting clusters running on Amazon Web Services (AWS) in Secure Token Service (STS) mode.
```

Comment:
I am not super sure about documenting this part. I think all it's doing with this info is the OLM related stuff. I worry that announcing it as a feature might make it seem like it can be used to do anything else (I have asked dev if it helps with any other task but want to make sure about that)

Reply:
OK great, was wondering that exactly as well, ty!

Reply:
https://github.com/openshift/openshift-docs/pull/66133#pullrequestreview-1675023716